vulcanai2/tests/models/test_dnn.py

```diff
@@ -97,3 +136,79 @@ def test_add_input_network(self, dnn_noclass, dnn_class):
         dnn_class.add_input_network(dnn_noclass)
         assert dnn_class.input_networks[dnn_noclass.name] is dnn_noclass
         assert dnn_class.in_dim == dnn_noclass.out_dim
+
+    # TODO: Failing fit! Fix the test
+    def test_fit_multi_input(self, multi_input_dnn,
+                             multi_input_train_loader,
+                             multi_input_test_loader):
+        """Test for fit function"""        
+        init_weights = copy.deepcopy(multi_input_dnn.network[0]._kernel.weight.detach())
+        multi_input_dnn_no_fit = copy.deepcopy(multi_input_dnn)
+        parameters1 = multi_input_dnn_no_fit.parameters()
+        try:
```

Comment:
Its failing the test here. probably it has something to do with how the dataloader is created. Any idea?

Reply:
You'll have to be more specific about the error your getting please

Reply:
resolved it in the latest commit